doc/source/reference/routines.emath.rst

```diff
@@ -1,9 +1,9 @@
-Mathematical functions with automatic domain (:mod:`numpy.emath`)
+Mathematical functions with automatic domain
 ***********************************************************************
 
 .. currentmodule:: numpy
 
-.. note:: :mod:`numpy.emath` is a preferred alias for :mod:`numpy.lib.scimath`,
+.. note:: :mod:`numpy.emath` is a preferred alias for ``numpy.lib.scimath``,
```

Comment:
revert this change?

Reply:
The link to `` :mod:`numpy.lib.scimath` `` is broken now because I changed the line at the bottom of this file from ``..automodule numpy.lib.scimath`` to ``..automodule numpy.emath``. Because of that, Sphinx generates references to members of `numpy.emath` which fixes the links but links to `numpy.lib.scimath` are now missing.

I was thinking of potentially adding the `numpy.lib.scimath` documentation but it seems there are no documents for the `numpy.lib.*` modules in `doc/source/reference`?  Because of that, I'm unsure if it should be documented/how to document it? What do you think?

I think there are a few other warnings (aside from `emath` related ones) also related to missing `numpy.lib.*` functions and modules.